src/main/java/uk/gov/hmcts/cmc/claimstore/services/JwtService.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.cmc.claimstore.services;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+
+@Component
+public class JwtService {
+
+    public boolean isCitizen(String authorisation) {
+        final DecodedJWT decodedToken = JWT.decode(authorisation.replace("Bearer ",""));
+        final String[] data = decodedToken.getClaims().get("data").asString().split(",");
+        return Arrays.stream(data).anyMatch("citizen"::equals);
```

Comment:
letter-holder is also a citizen action and letter-holder doesn't have the citizen role
REQ